### DISCUSSION TITLE
다중계정

### DISCUSSION THREAD

**Discussion by vork1541** [changdoc/naver-vaccine-macro#82] [Q&A]
안녕하세요 우선 이프로그램을 만들어주셔서 정말감사드립니다
한가지 궁굼한게 있는데 엣지 크롬 웨일 이렇게 브라우저 각각하나씩해서 엣지아이디1개 웨일1개 크롬1개 이렇게 아이디 3개를 동시에 돌릴수있나요? 하나는 네이버인증서 나머지 2개아이디는 휴대폰인증으로 돌릴생각입니다.

**Comment by changdoc** [comment]
아하, 계정이 나뉘어 있다면 가능할 것 같긴한데 안해봐서 모르겠습니다.
해보시고 후기 공유해주시면 좋겠네요~

**Comment by ghost** [comment]
일단 제가 해봤을 때에는 다른 아이디로 하면 계정밴 당하지 않는 것은 확인했습니다.

저는 크롬 / 크롬 시크릿탭(시크릿탭에서 하려면 확장 프로그램 내 설정에서 시크릿 모드에서 허용 체크해야합니다)

**Reply by soosooho** [reply]
아직 정상적으로 프로그램 돌아가고 있나요?? 병원은 3개 아이디 모두 같은곳인가요 다른곳인가요?

**Reply by ghost** [reply]
프로그램 정상작동합니다 저는한병원에 몰빵했습니다 새로고침 주기가2초니까 두아이디가 새로고침 텀이 겹치지않게 해서 성공했습니다.

**Reply by 3kwang** [reply]
계정 여러개를 같은 사람 인증서로 가능하가요??

**Reply by koala413** [reply]
다른 계정들은 핸드폰으로 인증받으시면 됩니다

**Reply by 3kwang** [reply]
> 다른 계정들은 핸드폰으로 인증받으시면 됩니다

웨일로 네이버 인증서 열고, 웨일시크릿모드로, 다른 아이디로 로그인 해서 휴대폰 인증하고 예약창 진입하니까 바로, '잠시후 시도하세요' 뜨던데 웨일하고 크롬 써야하나요??
크롬,크롬시크릿,웨일,웨일시크릿 4개 할 생각이였는데

**Reply by 3kwang** [reply]
> > 다른 계정들은 핸드폰으로 인증받으시면 됩니다
> 
> 웨일로 네이버 인증서 열고, 웨일시크릿모드로, 다른 아이디로 로그인 해서 휴대폰 인증하고 예약창 진입하니까 바로, '잠시후 시도하세요' 뜨던데 웨일하고 크롬 써야하나요??
> 크롬,크롬시크릿,웨일,웨일시크릿 4개 할 생각이였는데

아, 다시 테스트 해보니 되네요. 제가 뭘 잘못 눌렀거나 했나봐요. 4개 돌리려 했는데, 실명인증 최대 3개까지네요. 3개만 해야겠어요

**Comment by ghost** [comment]
일단 오늘오전 10시대에 성공했습니다 감사합니다~ 저는 웨일이랑 크롬으로 둘다 각각다른 아이디로 한병원을 지정하고 2초공백을 생각해서 새로고침이 중복되지않게 나눠서 시작을 눌렀습니다 지역은 경남입니다 개발자님 정말감사드립니다~

**Comment by ghost** [comment]
저도 금일 성공했습니다.

크롬 / 크롬 시크릿탭으로 하니까 2개가 타이밍이 시간 지나면 동기화가 되어버리는 것 같아서

크롬 / 웨일로 해서 성공했습니다 감사합니다 ^^

**Reply by soosooho** [reply]
저는 크롬/엣지 쓰는데 두개 다 2000ms 로 설정은 되어있지만 하나가 빠른지 새로고침 주기가 겹치다가 안겹치다가 하네요

**Reply by soosooho** [reply]
의미없는 질문이긴 한데 웨일로 잡았나요 크롬으로 잡았나요?
